crates/ruff_linter/src/rules/pycodestyle/rules/line_too_long.rs

```diff
@@ -7,25 +7,25 @@ use crate::rules::pycodestyle::overlong::Overlong;
 use crate::settings::LinterSettings;
 
 /// ## What it does
-/// Checks for lines that exceed the specified maximum character length.
+/// Checks for lines that exceed the specified maximum width.
 ///
 /// ## Why is this bad?
 /// Overlong lines can hurt readability. [PEP 8], for example, recommends
-/// limiting lines to 79 characters. By default, this rule enforces a limit
-/// of 88 characters for compatibility with Black, though that limit is
-/// configurable via the [`line-length`] setting.
+/// limiting lines to 79 characters. By default, this rule enforces a [display-width](http://www.unicode.org/reports/tr11/#Overview)
```

Comment:
Nit: remove dash in display-width? There's no dash in the linked source. (Also, do you mind changing the link to https?)